site/docs/getting-started/designing.mdx

```diff
@@ -5,11 +5,17 @@ hide_table_of_contents: true
 
 import JPMLogo from "@site/static/img/design-getting-started/jpmIcon.svg";
 import Globe from "@site/static/img/design-getting-started/globe.svg";
+import Assets1 from "@site/static/img/design-getting-started/assets1.png";
+import Assets2 from "@site/static/img/design-getting-started/assets2.png";
+import Assets3 from "@site/static/img/design-getting-started/assets3.png";
+import Assets4 from "@site/static/img/design-getting-started/assets4.png";
+import SelectIconImg from "@site/static/img/design-getting-started/SelectIconImg.png";
 import { InlineCard, CardProps } from "@site/src/components/card/Card";
 import Content from "@site/src/components/content/Content.tsx";
-
-import DesignCards, { SoloImgCard } from "./DesignCards";
-import IconsTable from "./IconsTable";
+import { PrintIcon } from "@salt-ds/icons";
+import { SaltProvider } from "@salt-ds/core";
+import DesignCards, { DesignCard } from "./_designing/DesignCards.tsx";
+import IconsTable, { DensityIcon } from "./_designing/IconsTable.tsx";
 
 import styles from "./designing.module.css";
```

Comment:
This doesn't seem to get used anywhere, so I think we can delete the import and the corresponding CSS file.

Reply:
A couple of classes from this file are still being used (`.strong` and `.colored`) but I removed the unused styles

Reply:
Ah, sorry. I searched the `.mdx` file for `styles.strong` and `styles.colored` and couldn't find any instances, but just realised Github was showing me the "preview" of the file, not the "raw" one.

Can you please move that CSS file into the `_designing/` folder though?